### ISSUE TITLE
Shovel?

### ISSUE THREAD

**Issue by johnasbaghi** [timothyrenner/tweetshovel#1]
It's great and all, but you might want to explain what you mean by shovel.

**Comment by timothyrenner** [comment]
You're right, I don't really explain the imagery. I think of each REST call as grabbing a shovel full of tweets from a pile. I'll put a blurb in the README.
